cli/src/main/java/keywhiz/cli/Printing.java

```diff
@@ -71,6 +63,42 @@ public void printClientWithDetails(Client client) {
     clientDetails.secrets.stream()
         .sorted(Comparator.comparing(SanitizedSecret::name))
         .forEach(s -> System.out.println(INDENT + SanitizedSecret.displayName(s)));
+
+    if (clientDetails.lastSeen == null) {
+      System.out.println("\tLast Seen: never");
+    } else {
+      String lastSeen = new SimpleDateFormat("MM/dd/yyyy HH:mm:ss").format(
+          new Date(clientDetails.lastSeen.toEpochSecond() * 1000));
+      System.out.printf("\tLast Seen: %s%n", lastSeen);
+    }
+
+    if (!clientDetails.description.isEmpty()) {
+      System.out.println("\tDescription:");
+      System.out.println(INDENT + clientDetails.description);
```

Comment:
Kinda weird we have a constant INDENT for "\t\t" but then a literal "\t" in the string above it -- again, I know you're not adding this pattern, but it is weird.